pkgs/build-support/trivial-builders.nix

```diff
@@ -1,5 +1,31 @@
-{ lib, stdenv, stdenvNoCC, lndir, runtimeShell }:
+{ lib, stdenv, stdenvNoCC, lndir, runtimeShell, zsh, dash, bash}:
 
+let
+  runtime = { 
+    preamble = "#!${runtimeShell}";
+    checkPhase = ''
+      ${stdenv.shell} -n $out
+    '';
+  };
+  bash = { 
```

Comment:
I wonder if not this could be removed, since `runtimeShell` variable is set to bash already.

Reply:
It's a bit superfluous, but I liked the distinction between the nixpkgs-wide runtime shell, whatever that may be, and bash. That way if I choose bash, I know it's bash, rather than assuming `runtimeShell == bash`. I know there was talk awhile ago of switching runtimeShell to dash for performance.